### PR TITLE
fix: Reskinning Input Widgets

### LINKED ISSUE #11023: [Enhancement] : Phone input widget reskinning
https://www.notion.so/appsmith/Phone-input-96ac4aa08ee04198a4a610524d0860ea

### LINKED ISSUE #11026: [Enhancement] : Currency input widget reskinning
Scope - https://www.notion.so/appsmith/Currency-widget-03bf484b3e474058aad34d6da4a0dac3

### LINKED ISSUE #11061: [Enhancement] : Input widget Reskinning
Scope - https://www.notion.so/appsmith/Input-Widget-4ac315b06a114083bda740731278f038

### PR DESCRIPTION
**Pull request by jsartisan**
Updates the UI for input widgets. Input widgets include:
1. Input widget
2. Currency input widget
3. Phone input widget

Fixes #11026 
Fixes #11023 
Fixes #11061 

## Type of change

- Bug fix 

## How Has This Been Tested?
- Manually tested by QA

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/widgets/BaseInputWidget/component/index.tsx [hunk 1]
```diff
@@ -203,7 +234,7 @@ const InputComponentWrapper = styled((props) => (
       margin: 0;
       .bp3-tag {
         background-color: transparent;
-        color: #5c7080;
+        color: red;
```
Comment (keyurparalkar) [inline app/client/src/widgets/BaseInputWidget/component/index.tsx:237]: Is it possible here to use this `red` color from the Colors constants?

File: app/client/src/widgets/CheckboxGroupWidget/index.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const CONFIG = {
   iconSVG: IconSVG,
   needsMeta: true,
   defaults: {
-    rows: 6,
+    rows: 4,
```
Comment (keyurparalkar) [inline app/client/src/widgets/CheckboxGroupWidget/index.ts:12]: Wouldn't changing this value cause UI inconsistencies between the existing users?

File: app/client/src/widgets/RadioGroupWidget/widget/index.tsx [hunk 3]
```diff
@@ -515,6 +515,7 @@ class RadioGroupWidget extends BaseWidget<RadioGroupWidgetProps, WidgetState> {
         loading={isLoading}
         onRadioSelectionChange={this.onRadioSelectionChange}
         options={isArray(options) ? compact(options) : []}
+        required={this.props.isRequired}
```
Comment (keyurparalkar) [inline app/client/src/widgets/RadioGroupWidget/widget/index.tsx:518]: If we are adding this as a new property, then I think we should right migrations for this.

**Comment by chandannkumar** [comment]
Tested this PR for enhancements of #11061 , #11023 , #11026 
- Input Widget Reskinning - Changes looks good and widget works fine without any breakage
- Currency Input widget - Changes LGTM 
- Phone Input widget - Changes LGTM

File: app/client/src/widgets/CheckboxGroupWidget/index.ts (hunk 2 above)
Comment (jsartisan) [reply app/client/src/widgets/CheckboxGroupWidget/index.ts]: Nope. The output will be same. its just that new value for rows won't have extra space.

File: app/client/src/widgets/BaseInputWidget/component/index.tsx (hunk 1 above)
Comment (jsartisan) [reply app/client/src/widgets/BaseInputWidget/component/index.tsx]: replaced it with css variable.

**Review by keyurparalkar — APPROVED** [review]
LGTM!

**Pull request merged**